tmva/tmva/CMakeLists.txt

```diff
@@ -76,7 +76,7 @@ endif()
 
 #---Handle BLAS dependent code. -----------------
 if(BLAS_FOUND AND imt)
-   message(STATUS "Using TMVA-DNN with BLAS installation") 
+   message(STATUS "Using TMVA-DNN with BLAS installation: ${BLAS_LIBRARIES}") 
```

Comment:
Does this print the path to the installed library (which I think would be very helpful when debugging!)

Reply:
Yes, it prints that.

And thanks for having seen the mixmcx file committed by mistake